src/Core/Utils/Attribs.hpp

```diff
@@ -115,9 +115,14 @@ class AttribHandle {
  * ...
  *
  * // somewhere else: access
- * auto iattribhandler = mng.getAttribHandler<float>("MyAttrib"); //  iattribhandler == inputfattrib
- * if (iattribhandler.isValid()) {
- *    auto &attrib = mng.getAttrib( iattribhandler );
+ * auto iattribhandler = mng.findAttrib<float>("MyAttrib"); //  iattribhandler == inputfattrib
+ * if (iattribhandler.isValid()) { // true
```

Comment:
This method is supposed to be removed, and replaced by `mng.isValid(iattribhandler)`.